components/avatar/index.css

```diff
@@ -10,107 +10,126 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-avatar.css";
+.spectrum-Avatar {
+  --spectrum-avatar-border-radius: 50%;
```

Comment:
_💭 Thought, not a necessary change:_

We could also set this equal to the component's width, like

`-spectrum-avatar-border-radius: var(--spectrum-avatar-size-100);`

Reply:
I like this suggestion!